Order.Management.PR/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Order.Management
+{
+    class Program
+    {
+        // Main entry
+        static void Main(string[] args)
+        {
+            var (customerName, address, dueDate) = CustomerInfoInput();
+
+            var orderedShapes = CustomerOrderInput();
+
+            InvoiceReport(customerName, address, dueDate, orderedShapes);
+
+            CuttingListReport(customerName, address, dueDate, orderedShapes);
+
+            PaintingReport(customerName, address, dueDate, orderedShapes);
+        }
+        
+        // Order Circle Input
+        public static Circle OrderCirclesInput()
+        {
+            Console.Write("\nPlease input the number of Red Circle: ");
```

Comment:
`Console.Write` is a static class, which means hard to test. We could create a ConsoleUserInterface class that implements the `IUserInterface` interface and wraps a call to the `Console.Write` method inside an instance method called `Write`.